webrtc.html

```diff
@@ -11573,9 +11573,89 @@ <h2 id="sec.idp-error-handling">IdP Error Handling</h2>
         <code>RTCError</code> constructor to create and initialize
         the subclass instance with an [[\ErrorData]] internal slot.</p>
         <section>
-      <h2>TODO errorDetailEnum</h2>
-      <p>TODO - make a enum for all the error details? Is the enum OK for
-      extending?<p>
+          <h2><dfn>RTCErrorDetailType</dfn> Enum</h2>
+          <div>
+          <pre class="idl">enum RTCErrorDetailType {
+              "data-channel-failure",
+              "dtls-failure",
+              "idp-bad-script-failure",
+              "idp-execution-failure",
+              "idp-load-failure",
+              "idp-need-login",
+              "idp-timeout",
+              "idp-tls-failure",
+              "idp-token-expired",
+              "idp-token-invalid",
+              "parameters-error",
+              "sctp-failure",
```

Comment:
Added a comment for parameters-error and dtls-failure.  sctp-failure and data-channel-failure are defined in PR https://github.com/w3c/webrtc-pc/pull/1011 .